packages/ui-lib/src/Pages/molecules/PageHeader.tsx

```diff
@@ -1,28 +1,78 @@
-import React from 'react';
+import React, { ReactElement } from 'react';
 import styled from 'styled-components';
-
-import PageTitle from '../atoms/PageTitle';
+import PageTitle, { IIconPos } from '../atoms/PageTitle';
 import IntroductionText from '../atoms/IntroductionText';
+import Tag from '../../Misc/atoms/Tag';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: space-between;
+  gap: 32px;
+`;
+
+const LeftPanel = styled.div`
+`;
+
+const RightPanel = styled.div`
+`;
 
-const Container = styled.div``;
+const TagListWrapper = styled.div`
+  display: inline-flex;
+  align-items: flex-start;
+  flex-wrap: wrap;
+  gap: 4px;
+  margin-bottom: 5px;
+  margin-top: 29px;
+`;
+
+export type IHeaderTag = {
+  label: string
+  linkTo?: string
+  icon?: string
+}
 
 interface IProps {
   title: string
   areaHref?: string
   areaTitle?: string
   icon?: string
+  iconPosition?: IIconPos
   introductionText?: string
   updateDocTitle?: boolean
-  hideAreaInDocTitle? : boolean
+  hideAreaInDocTitle?: boolean
+  tagList?: IHeaderTag[]
+  areaTitleBottom?: boolean
+  customRight?: React.ReactNode | React.FC | ReactElement;
```

Comment:
For certain layouts previously I believe I've used `[place]Content` for this kind of prop. In the SplitLayouts I used the naming convention `[place]Area` as it's a more nested thing.

Could we unify the naming with one of these for consistency? Thank you

Reply:
I'll update to rightContent